frontend/www/js/omegaup/components/common/Paginator.vue

```diff
@@ -19,11 +25,22 @@
 <script lang="ts">
 import { Vue, Component, Prop } from 'vue-property-decorator';
 import { omegaup, T } from '../../omegaup';
+import { types } from '../../api_types';
 
 @Component
 export default class Paginator extends Vue {
-  @Prop() pagerItems!: omegaup.Paginator[];
+  @Prop() pagerItems!: types.PageItem[];
+  @Prop({ required: false }) callback!: Function;
```

Comment:
```suggestion
  @Prop({ required: false }) callback!?: Function;
```

esto funciona? así puedes checar `!!callback` en vez de `requireCallback`. no estoy seguro entre el orden del `!` y el `?`.

Reply:
Si funciona, pero ya no es necesario el `?`, con el `{ required: false }` es suficiente.